site2/docs/reference-metrics.md

```diff
@@ -0,0 +1,234 @@
+in `bookkeeper.conf` configuration file.
+
+### Server Metrics
+
+| Name | Type | Description |
+|---|---|---|
+| bookie_SERVER_STATUS | Gauge | The server status for bookie server. <br><ul><li>1: the bookie is running in writable mode.</li><li>0: the bookie is running in readonly mode.</li></ul> |
+| bookkeeper_server_ADD_ENTRY_count | Counter | The total number of ADD_ENTRY requests received at the bookie. Label `success` used to distinguish successes and failures |
+| bookkeeper_server_READ_ENTRY_count | Counter | The total number of READ_ENTRY requests received at the bookie. Label `success` used to distinguish successes and failures |
+| bookie_WRITE_BYTES | Counter | The total number of bytes written to the bookie |
+| bookie_READ_BYTES | Counter | The total number of bytes read from the bookie |
+| bookkeeper_server_ADD_ENTRY_REQUEST | Histogram | The histogram of request latency of ADD_ENTRY requests at the bookie. Label `success` used to distinguish successes and failures | 
+| bookkeeper_server_READ_ENTRY_REQUEST | Histogram | The histogram of request latency of READ_ENTRY requests at the bookie. Label `success` used to distinguish successes and failures | 
+
+### Journal Metrics
+
+| Name | Type | Description |
+|---|---|---|
+| bookie_journal_JOURNAL_SYNC_count | Counter | The total number of journal fsync operations happening at the bookie. Label `success` used to distinguish successes and failures |
+| bookie_journal_JOURNAL_QUEUE_SIZE | Gauge | The total number of requests pending in the journal queue. |
+| bookie_journal_JOURNAL_FORCE_WRITE_QUEUE_SIZE | Gauge | The total number of force write (fsync) requests pending in the force-write queue. |
+| bookie_journal_JOURNAL_CB_QUEUE_SIZE | Gauge | The total number of callbacks pending in the callback queue. |
+| bookie_journal_JOURNAL_ADD_ENTRY | Histogram | The histogram of request latency of adding entries to the journal. |
+| bookie_journal_JOURNAL_SYNC | Histogram | The histogram of fsync latency of syncing data to the journal disk. |
+
+### Storage Metrics
+
+| Name | Type | Description |
+|---|---|---|
+| bookie_ledgers_count | Gauge | The total number of ledgers stored in the bookie |
+| bookie_entries_count | Gauge | The total number of entries stored in the bookie |
+| bookie_write_cache_size | Gauge | The bookie write cache size (in bytes) |
+| bookie_read_cache_size | Gauge | The bookie read cache size (in bytes) |
+| bookie_DELETED_LEDGER_COUNT | Counter | The total number of ledgers deleted since the bookie starts |
+| bookie_ledger_writable_dirs | Gauge | The number of writable directories in the bookie |
+
+## Broker
+
+The broker metrics are exposed under "/metrics" at port 8080. You can change the port by updating `webServicePort` to a different port
+in `broker.conf` configuration file.
+
+All the metrics exposed by broker are labelled with `cluster=${pulsar_cluster}`. The value of `${pulsar_cluster}` is the pulsar cluster
+name you configured in `broker.conf`.
+
```

Comment:
Consider adding the following to show a preview/structure of long contents? 
Besides, it is easy for users to locate info for #### (heading 4) contents which are not shown on the right TOC on the webpage.

If agree, please check all cases.

```
Broker has the following kinds of metrics:

* [Namespace metrics](#namespace-metrics)
    * [Replication metrics](#replication-metrics)
* [Topic metrics](#topic-metircs)
* [Subscription metrics](#subscription-metrics)
* [Consumer metrics](#consumer-metrics)
```